### PR TITLE
feat: Add model registry tests

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/registry/event/event_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	modeljobsv1alpha1 "github.com/kleveross/klever-model-registry/pkg/apis/modeljob/v1alpha1"
+	modeljobfake "github.com/kleveross/klever-model-registry/pkg/clientset/clientset/versioned/fake"
+	"github.com/kleveross/klever-model-registry/pkg/registry/client"
+	"github.com/kleveross/klever-model-registry/pkg/registry/event"
+	"github.com/kleveross/klever-model-registry/pkg/registry/modeljob"
+)
+
+var _ = Describe("Event", func() {
+	client.KubeMainClient = testclient.NewSimpleClientset()
```
Comment (gaocegege) [inline pkg/registry/event/event_test.go:17]: Prefer k8sfake here since we are using modeljobfake below.

Comment (simon-cj) [reply pkg/registry/event/event_test.go:17]: comment are addressed, PTAL

**Comment by simon-cj** [comment]
> I suggest setting up Travis CI first and run the test in CI.

Add a simple travis CI, PTAL

File: go.mod [hunk 2]
```diff
@@ -6,6 +6,8 @@ require (
 	github.com/astaxie/beego v1.12.2 // indirect
 	github.com/caicloud/nirvana v0.2.5-0.20200717071154-a7421a7b084c
 	github.com/go-logr/logr v0.1.0
+	github.com/golang/mock v1.2.0
+	github.com/kubernetes/dashboard v1.10.1
```
Comment (gaocegege) [inline go.mod:10]: Why do we need dashboard?

File: pkg/registry/event/event_test.go (hunk 1 above)
Comment (gaocegege) [reply pkg/registry/event/event_test.go:17]: I mean 
```suggestion
	client.KubeMainClient = k8sfake.NewSimpleClientset()
```

Comment (simon-cj) [reply pkg/registry/event/event_test.go]: oh, add and push is error, I should check it, now PTAL

File: go.mod (hunk 2 above)
Comment (simon-cj) [reply go.mod]: It removed， we don't need dashboard， it is trial before

**Pull request merged**